grype/presenter/models/document.go

```diff
@@ -37,7 +37,7 @@ func NewDocument(packages []pkg.Package, context pkg.Context, matches match.Matc
 	metadataProvider vulnerability.MetadataProvider, appConfig interface{}, dbStatus interface{}) (Document, error) {
 	// we must preallocate the findings to ensure the JSON document does not show "null" when no matches are found
 	var findings = make([]Match, 0)
-	for m := range matches.Enumerate() {
+	for _, m := range matches.Sorted() {
```

Comment:
is there going to be a situation where we wouldn't want sorted packages by default? Why does it need to be explicitly called out?